### ISSUE TITLE
Could not load file or assembly 'Plugin.Connectivity, Version=4.0.167.0

### ISSUE THREAD

**Issue by aherrick** [jamesmontemagno/ConnectivityPlugin#123]
Updated from `4.0 beta139` to `4.0 beta167` and receive the following error in Forms UWP.

```
System.IO.FileLoadException occurred
  HResult=0x80131040
  Message=Could not load file or assembly 'Plugin.Connectivity, Version=4.0.167.0, Culture=neutral, PublicKeyToken=null'. The located assembly's manifest definition does not match the assembly reference. (Exception from HRESULT: 0x80131040)
  Source=MyApp
  StackTrace:
   at Windows.UI.Xaml.Controls.Frame.Navigate(Type sourcePageType, Object parameter)
   at AXLE_Driver.UWP.App.OnLaunched(LaunchActivatedEventArgs e) in C:\Dropbox\Projects\BHS\Driver Mobile\AXLE_Driver\AXLE_Driver.UWP\App.xaml.cs:line 115
```

**Comment by jamesmontemagno** [comment]
update all of your nugets

**Comment by aherrick** [comment]
They are all updated. 

Still seeing in `4.0.169`
No issue in `4.0.139`

**Comment by jamesmontemagno** [comment]
Ah, i see it, fixed in next 4.0. doesn't effect 3.x

**Comment by aherrick** [comment]
Fixed in `4.0.171`

Thanks!
